tests/intentManager.js

```diff
@@ -190,5 +196,82 @@ define([
                 });
             });
         });
+
+        describe("custom intents", function() {
+            var vellum, mug;
+            before(function(done) {
+                util.init({
+                    intents: {templates: templates},
+                    features: {
+                        rich_text: false,
+                        templated_intents: true,
+                        custom_intents: true,
+                    },
+                    core: {
+                        onReady: function () {
+                        vellum = this;
+                        mug = util.addQuestion("AndroidIntent", "intent");
+                        util.clickQuestion("intent");
+                        done();
+                    }}
+                });
+            });
+
+            it("should always have one custom option", function() {
```

Comment:
I'd add a test that the input disabling happens correctly (readonly on selecting custom, writable on selecting anything else).